unlock-app/package.json

```diff
@@ -3,19 +3,24 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.1.2",
```

Comment:
FYI, have generally found that locking in version numbers (by removing carat) saves a lot of hassle later when a random package will upgrade itself on a deployment and cause unknown issues.

Reply:
Move fast, break things!